<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
<pkgmetadata>
	<maintainer type="person">
		<email>gentoo@houseofsuns.org</email>
		<name>Markus Walter</name>
	</maintainer>
	<longdescription># python-supporterhttps://pypi.org/project/python-supporter&lt;pre&gt;pip install python-supporter&lt;/pre&gt;```from python_supporter import logginglogging.basicConfig(logging.ERROR)#logging.basicConfig(logging.INFO)#logging.basicConfig(logging.DEBUG)#logging.basicConfig()#logging.basicConfig(logging.ERROR, filename='log.txt')#logging.basicConfig(logging.INFO, filename='log.txt')#logging.basicConfig(logging.DEBUG, filename='log.txt')#logging.basicConfig(filename='log.txt')logging.error('This is error message') #2023-03-19 22:36:47: ERROR: This is error messagelogging.info('This is info message')logging.debug('This is debug message')``````import python_supportertext = python_supporter.file.read_file(file)python_supporter.file.write_file(file, text)``````import python_supporterchrome_debugger_address = &quot;127.0.0.1:1001&quot;li = chrome_debugger_address.split(&quot;:&quot;)remote_debugging_port = int(li[1])remote_debugging_address = li[0]port_opened = python_supporter.socket.check_port_open(remote_debugging_address, remote_debugging_port)```</longdescription>
</pkgmetadata>